style.css

```diff
@@ -0,0 +1,634 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  background: #fff;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+button:enabled {
+  color: #6070ff;
+  background-color: #fff;
+  border-color: #6070ff;
+}
+
+button:hover {
+  color: #fff;
+  background-color: #6070ff;
+}
+
+button:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+button:disabled {
+  color: #5e6c84;
+  border-color: #5e6c84;
+  background-color: #fff;
+}
+
+.navibar {
+  background: #ffff;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.nav-list {
+  display: none;
+}
+
+.navibar .nav-container {
+  place-content: space-between;
+  display: flex;
+}
+
+.navibar .logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.h-container {
+  background-image: url("./img/header-shapes-mobile.png"), url("./img/header-background-mobile.png");
+  background-repeat: no-repeat, no-repeat;
+  background-size: cover, 100% 100%;
+  padding-top: 114px;
+  padding-bottom: 114px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.first-text {
+  position: static;
+  width: 60%;
+  left: 24px;
+  top: 114px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 28px;
+  line-height: 42px;
+  display: flex;
+  color: #172b4d;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0;
+  align-items: center;
+}
+
+h2 {
+  position: static;
+  width: 60%;
+  left: 24px;
+  top: 114px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 28px;
+  line-height: 42px;
+  display: flex;
+  color: #172b4d;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0;
+  align-items: center;
+}
+
+.second-text {
+  position: static;
+  width: 327px;
+  height: 146px;
+  left: 24px;
+  top: 276px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.icon {
+  position: static;
+  left: 0;
+  top: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0;
+}
+
+.h-container .icon-img {
+  padding-left: 0;
+  justify-content: flex-start;
+  gap: 28px;
+  display: flex;
+  flex-direction: row;
+  margin-top: 13px;
+}
+
+.projects {
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  width: 100%;
+  padding: 100px 0;
+  background-color: #f7f7f9;
+}
+
+.card-contain {
+  border: 1px solid #7a869a;
```

Comment:
- [ ] Kindly add the right border color `#DFE1E6` for each card 👍🏿

Reply:
- [ ] Optional: Kindly make sure to display the cards correctly as you can see all the cards in even numbers like 2 and 4 follow row reverse as the content is first and then the image. As the previous reviewer forgot to mention this. I have kept it optional but I recommend you implement it in your free time.